test/sanity/pylint/ignore.txt

```diff
@@ -64,6 +64,7 @@ lib/ansible/modules/storage/infinidat/infini_host.py ansible-format-automatic-sp
 lib/ansible/modules/storage/infinidat/infini_vol.py ansible-format-automatic-specification
 lib/ansible/modules/storage/purestorage/purefa_host.py ansible-format-automatic-specification
 lib/ansible/modules/storage/purestorage/purefa_pg.py ansible-format-automatic-specification
+lib/ansible/modules/storage/emc/emc_vnx_sg_member.py ansible-format-automatic-specification
```

Comment:
I would not add this line as this file exists for existing modules only.

Reply:
Without this entry, i'll get pylint errors because i'm using automatic formatting instead of numbered.  But if support for python 2.6 is still required i'll use numbers in formatting.